icu4j/main/classes/core/src/com/ibm/icu/message2/Mf2ParseException.java

```diff
@@ -0,0 +1,73 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.message2;
+
+/**
+ * An exception thrown when parsing a {@link MessageFormatter} syntax fails.
+ *
+ * @internal ICU 72 technology preview
+ * @deprecated This API is for ICU internal use only.
+ */
+@Deprecated
+public class Mf2ParseException extends RuntimeException {
+
+    private static final long serialVersionUID = -4580228572386715579L;
+
+    /**
+     * {@inheritDoc}
+     *
+     * @internal ICU 72 technology preview
+     * @deprecated This API is for ICU internal use only.
+     */
+    @Deprecated
+    public Mf2ParseException() {
+        super();
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @internal ICU 72 technology preview
+     * @deprecated This API is for ICU internal use only.
+     */
+    @Deprecated
+    public Mf2ParseException(String message) {
+        super(message);
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @internal ICU 72 technology preview
+     * @deprecated This API is for ICU internal use only.
+     */
+    @Deprecated
+    public Mf2ParseException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @internal ICU 72 technology preview
+     * @deprecated This API is for ICU internal use only.
+     */
+    @Deprecated
+    public Mf2ParseException(Throwable cause) {
+        super(cause);
+    }
+
+    /**
+     * {@inheritDoc}
+     *
+     * @internal ICU 72 technology preview
+     * @deprecated This API is for ICU internal use only.
+     */
+    @Deprecated
+    protected Mf2ParseException(String message, Throwable cause,
+                               boolean enableSuppression,
+                               boolean writableStackTrace) {
```

Comment:
Same question as for the other exception, about whether we need/want to override this one.

Reply:
The Mf2ParseException is removed.

But otherwise the same answer: inheriting from `ICUException` means I don't need to override this,
and adding it to `ICUException` is a decision that is not part of MF2.